_maps/map_files/Deltastation/DeltaStation2.dmm

```diff
@@ -23718,6 +23715,11 @@
 /obj/machinery/light_switch{
 	pixel_x = 23
 	},
+/obj/machinery/computer/cargo/express{
+	tag = "icon-computer (WEST)";
```

Comment:
This var shouldn't be here, probably just want to delete it manually.

Reply:
yeah i was wondering where that was from.